adserver/forms.py

```diff
@@ -746,6 +746,57 @@ class Meta:
         }
 
 
+class AdvertisementCopyForm(forms.Form):
+
+    """Used by advertisers to re-use their ads."""
+
+    advertisements = AdvertisementMultipleChoiceField(
```

Comment:
Copying multiple ads is a huge workflow improvement. This will change how I ads across flights 💯